recipes/celero/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

Reply:
Actually 3.0.2 is the min version of celero: https://github.com/DigitalInBlue/Celero/blob/83b592b134cb41e2e5611714bce0bf61413eb12b/CMakeLists.txt#L24
But I could have set it to 2.8.11, since maximum version wins.

Reply:
Hmmm, there's no rule about what this value should be, I guess this comment should be resolved

Reply:
Yes, there is no rule for recipe/xxx/all/CMakeLists.txt. Usually we try to keep 2.8, but there is no problem following upstream version (you just need to spend more time checking). But for test_package/CMakeLists.txt we agree on 3.1 at least. But why such difference? Most users build Conan packages running conan install command, not downloading the recipe and running test package.